clearpath_generator_common/clearpath_generator_common/param/platform.py

```diff
@@ -236,6 +236,28 @@ def generate_parameters(self, use_sim_time: bool = False) -> None:
                                     ],
                                     'contains': ['MCU']}}}}})
 
+            # Add cooling for A300 only for now
+            if self.clearpath_config.get_platform_model() == Platform.A300:
+                self.param_file.update(
+                    {self.DIAGNOSTIC_AGGREGATOR_NODE: {
+                        'platform': {
+                            'analyzers': {
+                                'cooling': {
+                                    'type': 'diagnostic_aggregator/GenericAnalyzer',
+                                    'path': 'Cooling',
+                                    'contains': ['Fan', 'Thermal']}}}}})
```

Comment:
Instead of a mess of closing braces, brackets, and parentheses all on one line, can we stagger these so they're lined up with their openings?